ui/packages/catalog-ui-search/src/main/webapp/component/filter-builder/filter-serialization.js

```diff
@@ -68,6 +68,11 @@ const transformFilter = filter => {
   const { type, property } = filter
 
   const value = CQLUtils.isGeoFilter(filter.type) ? filter : filter.value
+  if (type === 'DURING' && filter.value.includes('/')) {
+    const dates = filter.value.split('/')
+    filter.from = dates[0] || null
+    filter.to = dates[1] || null
+  }
```

Comment:
:pencil2: I think it would be better to move this logic down here: https://github.com/codice/ddf/pull/5451/files#diff-959422c3c1e9582f5e2903d26b2e9ed1R108